src/components/navigation-dropdown/navigation-dropdown.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ControlSelect from '@mapbox/react-control-select';
+// import { routeToPrefixed } from '@mapbox/batfish/modules/route-to';
+
+class NavigationDropdown extends React.Component {
+  render() {
+    const { props } = this;
+    const optionsArray = props.dropdownOptions.map(option => {
+      return {
+        label: option.title,
+        value: option.path
+      };
+    });
+    const currentOption = optionsArray.filter(option => {
+      return props.currentPath === option.value;
+    })[0];
+
+    return (
+      <div className="py24 px24">
```

Comment:
Not sure about this padding, since I think that layout should be determined by the parent element.